tools/topology/topology2/cavs/cavs-sdw-src-gain-mixin.conf

```diff
@@ -0,0 +1,180 @@
+<searchdir:cavs>
+<searchdir:include>
+<searchdir:include/common>
+<searchdir:include/components>
+<searchdir:include/components/cavs>
+<searchdir:include/dais>
+<searchdir:include/controls>
+<searchdir:include/pipelines/cavs>
+
+<vendor-token.conf>
+<tokens.conf>
+<virtual.conf>
+<src-gain-mixin-playback.conf>
+<mixout-gain-dai-copier-playback.conf>
+<passthrough-capture.conf>
+<passthrough-be.conf>
+<data.conf>
+<pcm.conf>
+<pcm_caps.conf>
+<fe_dai.conf>
+<alh.conf>
+<hw_config.conf>
+<manifest.conf>
+<route.conf>
+<cavs/common_definitions.conf>
+<copier.conf>
+<pipeline.conf>
+<dai.conf>
+<host.conf>
+
+Object.Base.manifest."sof_manifest" {
+	Object.Base.data."SOF ABI" {
+		bytes	"0x03,0x00,0x16,0x00,0x01,0x00"
+	}
+}
+
+#
+# List of all DAIs
+#
+#ALH Index: 0, Direction: duplex
+Object.Dai {
+	ALH."2" {
+		id 		2
+		direction	"playback"
+		name		SDW2-Playback
```

Comment:
SDW2-Playback and SDW3-Capture, are this device specific? Which device are you testing with?

Reply:
ANDREWS

Reply:
Adjust it

Reply:
I know we need to change this and doing it. My point was cavs-sdw.conf has SDW0-Playback and SDW0-Capture, we should match this for consistency.